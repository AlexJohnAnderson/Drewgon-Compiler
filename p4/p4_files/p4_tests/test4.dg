int main(){
	int b;
	b = true;
	return b;
}
