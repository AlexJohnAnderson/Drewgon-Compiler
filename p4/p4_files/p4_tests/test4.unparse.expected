int main(->int)(){
    int b(int);
    b(int) = true;
    return b(int);
}